docs/topics/tour/kotlin-tour-intermediate-extension-functions.md

```diff
@@ -86,8 +86,10 @@ fun HttpClient.post(url: String): HttpResponse = request("POST", url, emptyMap()
 ```
 {validate="false"}
 
-These `.get()` and `.post()` functions call the `request()` function with the correct HTTP method, so you don't have to.
-They streamline your code and make it easier to understand:
+These `.get()` and `.post()` functions extend the `HttpClient` class. Because they're called on an instance of the `HttpClient` class
+as the receiver, they can directly use the `request()` function from the `HttpClient` class. These extension functions handle
+the common cases for you by calling the `request()` function with the appropriate HTTP method. They simplify your code 
```

Comment:
I would suggest going second person here, because the current version may imply too much agency for functions: "You can use these extension functions to [conveniently?] call the `request()` function with the appropriate HTTP method. This way you simplify your code and make it easier to understand."